server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -302,6 +302,12 @@ public void run() {
               }
             }
 
+            try {
+              nextLog.offer(t, 12, TimeUnit.HOURS);
+            } catch (InterruptedException ex) {
+              // ignore
+            }
+
```

Comment:
It looks like this was added back but just in a different place?

Reply:
The code was restructured from one try/catch block into two try/catch blocks.  The original try/catch had this, but the two new ones did not.